Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -128,6 +154,17 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		044B0BC01F7D8B000023D0A9 /* Embed Frameworks */ = {
```

Comment:
Is this change intended?

Reply:
Yes, I added a new target for MC Build, each target requires this framework to build the app.

Reply:
Right and they all stack up in the same project file.